plugins/slack_app/app/models/slack_message.rb

```diff
@@ -0,0 +1,116 @@
+class SlackMessage
+  def initialize(deploy)
+    @deploy = deploy
+  end
+
+  def deliver
+    url = DeployResponseUrl.find_by(deploy_id: @deploy.id).try(:response_url)
+    Faraday.new(url).post do |req|
```

Comment:
Faraday.new with nil sounds like it would give weird errors ... blow up nicely instead ?